### PR TITLE
Feature/moov 2297 context menu

### PR DESCRIPTION
**Pull request by donalnofrixion**

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/77.

**Review by sipsorcery — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/77.

File: packages/components/src/components/ui/molecules/ConnectedAccountContextMenu/ConnectedAccountContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          align="center"
+          className="mx-4 bg-white rounded-lg shadow-[0px_0px_8px_rgba(4,_41,_49,_0.1)] py-4 pl-6 pr-6"
+        >
+          <DropdownMenuItem
+            key="refresh-menu"
+            onSelect={(event) => {
+              event.stopPropagation()
+              onRenewConnection && onRenewConnection(account)
+            }}
+            onClick={(event) => event.stopPropagation()}
```
Comment (Publicker) [inline packages/components/src/components/ui/molecules/ConnectedAccountContextMenu/ConnectedAccountContextMenu.tsx:45]: What's the difference between `onSelect` and `onClick`? I've noticed we use `stopPropagation` for both. What do you think about integrating `event.stopPropagation()` directly into `DropdownMenuItem`?

File: packages/components/src/components/ui/atoms/DropDown/DropDown.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as DropdownMenuPrimitive from '@radix-ui/react-dropdown-menu'
 import * as React from 'react'
 
-import { cn } from '../../lib/utils/utils'
+import { cn } from '../../../../../../../apps/business/src/lib/utils/utils'
```
Comment (Publicker) [inline packages/components/src/components/ui/atoms/DropDown/DropDown.tsx:4]: Using `cn` from business in components seems wrong. I'd suggest to import it directly from the same project.

**Review by Publicker — CHANGES_REQUESTED** [review]
In addition to commenting on the code, I'll do my best to provide a comprehensive review, especially in Pablo's absence. Those are some big shoes to fill! (so I'll try my best).

- The 'Renew Connection' modal doesn't match the design. Comments here:
Screenshot 2023-09-25 at 10 11 10



- The context menu modal doesn't open as smoothly as it's usual for the business portal. I believe it lacks the animation. It'd be awesome if it had the same or a similar animation to the one in the Payment Request table. Here's a video showcasing its current opening and closing:

https://github.com/nofrixion/nofrixion.business/assets/52673485/ca817e0a-6bd0-43be-9464-4b327fccb4ef

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/77.

**Review by Publicker — CHANGES_REQUESTED** [review]
Animation is really nice! And spacing looks better. However, I noticed a few things:

- Regarding spacing and sizes:
#77 - Review 2

- I believe the Privacy Policy should directly link to https://nofrixion.com/privacy-policy and open in a new tab.

- The changes to the dropdown have made the "Revoke" icon appear a bit off - almost like it's cropped or of lower quality.
Revoke icon feedback

- When cycling through the dropdown options ("Refresh" and "Revoke connection") with the keyboard, it doesn't trigger the "hover" state, unlike what happens in the Payment Request table. But it works fine with a regular hover.

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/77.

**Pull request merged**